src/documentation/runtime/debugging.page

```diff
@@ -72,7 +72,31 @@ Orocos.run 'my_project::Task' => 'task', 'my_deployment',
 end
 ~~~
 
+
 in which case the valgrind output is redirected to a separate process logfile,
 which is the process' log file name with .valgrind appended (in the example
 above, task-PID.log.valgrind and my_deployment-PID.log.valgrind, where PID is
 the process ID of the started deployment)
+
+It is also possible to pass options to valgrind:
+
+~~~ ruby
+Orocos.run 'my_project::Task' => 'task', 'my_deployment',
+    :valgrind => true, :valgrind_options => ['--undef-value-errors=no'] do
+   ...
+end
+~~~
+
+Ignore RTT Exception Handling
+--------
+RTT catches by default all exeption from the tasks. 
+This is because RTT tries to be as safe as possible.
+A somehow crashed task should never influence the execution of
+other components within the same compositon. However for debugging
+it is usefull to let a component raise the exception to cause a crash. 
```

Comment:
However, for debugging purposes, it is convenient to let the exceptions be unhandled and cause a process abort.